cfg/pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+---
+repos:
+-
+  hooks:
+  -
+    id: 'hadolint-docker'
+    files: 'rust/Dockerfile'
+  repo: 'https://github.com/hadolint/hadolint'
+  rev: 'v2.12.0'
+-
+  hooks:
+  -
+    args:
```

Comment:
How about `yamllint` using the same config file as we have in `emojicoin-dot-fun`?

Reply:
Bad idea IMO. they have their own yamls, and that will create a ton of diff. Also, we literally just have one yaml file here and it's this one.